### PR TITLE
Checking item's media type - fix condition

### PR DESCRIPTION
**Pull request by burekas7**

### CODE REVIEW

File: service.py [hunk 1]
```diff
@@ -119,14 +119,28 @@ def mirror_sub(id, filename, sub_file):
         pass
 
 def takeTitleFromFocusedItem():
-    labelType = xbmc.getInfoLabel("ListItem.DBTYPE")  #movie/tvshow/season/episode
     labelMovieTitle = xbmc.getInfoLabel("ListItem.OriginalTitle")
     labelYear = xbmc.getInfoLabel("ListItem.Year")
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
     isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```
Comment (maortal) [inline service.py]: If you moved labelType the condition here is now broken

File: service.py [hunk 2]
```diff
@@ -119,14 +119,28 @@ def mirror_sub(id, filename, sub_file):
...
     isItEpisode = xbmc.getCondVisibility("Container.Content(episodes)") or labelType == 'episode'
```
Comment (maortal) [inline service.py]: Here also..

File: service.py (hunk 1 above)
Comment (burekas7) [inline service.py]: Why broken?
See the whole file, every thing is the same, I just move it down few lines and change the condition as CaTz told me.

Comment (maortal) [inline service.py]: The or condition is now comparing to an unknown variable

Comment (burekas7) [inline service.py]: OH, sorry, my mistake, I forgot remove it.
Thanks.

**Pull request closed without merging**